### DISCUSSION TITLE
Using pseudocode.js

### DISCUSSION THREAD

**Discussion by qlinhta** [alshedivat/al-folio#2327] [Q&A] [answered]
Hi,

I was trying to find proper way to presents our pseudo-code, using pseudocode.js might be the good idea but I was not successful. If anyone has done this successfully, I would love to have your guides or suggestions.

Thank you so much.

**Comment by george-gca** [comment]
Hummm it seems like it is easy to integrate it, but not in a general way. I can take a look at this in a couple of days, to check how hard it would be. Meanwhile it is possible to use it like recommended [in their example](https://saswat.padhi.me/pseudocode.js/).

**Comment by george-gca** [comment, marked as answer]
Implemented this in #2344.
